### PR TITLE
restore drools-osgi for features not depending on camel

### PR DESCRIPTION
**Pull request by mariofusco**

### CODE REVIEW

File: drools-osgi/drools-karaf-features/pom.xml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  <properties>
+    <!--
+      CONVENTIONS:
+      - All properties used in features.xml are declared here with the prefix "karaf.".
+      - If a property relates to a version property in kie-parent-with-dependencies or jboss-integration-platform-bom,
+        then it must follow the pattern "karaf.[servicemix.]{relatedVersionPropertyName}".
+    -->
+    <!-- TODO we should use the karaf features-maven-plugin -->
+    <!-- Normal reuses -->
+    <karaf.version.org.apache.camel>${version.org.apache.camel}</karaf.version.org.apache.camel>
```
Comment (psiroky) [inline drools-osgi/drools-karaf-features/pom.xml]: I think we should not reference the camel version here. It is not used in the features file.

File: drools-osgi/drools-karaf-features/pom.xml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    <karaf.version.org.apache.cxf>${version.org.apache.cxf}</karaf.version.org.apache.cxf>
```
Comment (psiroky) [inline drools-osgi/drools-karaf-features/pom.xml]: The same as the camel version above.

File: drools-osgi/drools-karaf-features/src/main/filtered-resources/repository/features.xml [hunk 3]
```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="camel-drools-example-${project.version}"
```
Comment (psiroky) [inline drools-osgi/drools-karaf-features/src/main/filtered-resources/repository/features.xml]: This is something I noticed by accident the other day -- the name should be something like `drools-${project.version}` (or probably better to use `drools-${version.org.kie}`). We need to fix this on master as well.

File: drools-osgi/drools-karaf-features/src/main/filtered-resources/repository/features.xml [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+          xmlns="http://karaf.apache.org/xmlns/features/v1.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.0.0 http://karaf.apache.org/xmlns/features/v1.0.0">
+
+  <feature name="drools-common" version="${version.org.drools}" description="Drools Commons">
+    <feature version="[2.6,3.0)">cxf-specs</feature>
```
Comment (psiroky) [inline drools-osgi/drools-karaf-features/src/main/filtered-resources/repository/features.xml]: I have not idea about this one -- but do we need to depend on CXF for Drools?

**Comment by mariofusco** [comment]
Merged manually

**Pull request closed without merging**